src/meltano/cli/interactive/config.py

```diff
@@ -0,0 +1,408 @@
+        if unexpanded_value:
+            current_value = unexpanded_value if unexpanded_value is not None else ""
+            details.add_row(Text("Current Expanded Value"), Text(f"{expanded_value}"))
+        else:
+            current_value = value if value is not None else ""
+        details.add_row(
+            Text(f"Current Value ({label})"),
+            Text.from_markup(f"[{VALUE_COLOR}]{current_value}[/{VALUE_COLOR}]"),
+        )
+        # setting kind
+        if setting_def.kind:
+            details.add_row(Text("Kind"), Text(f"{setting_def.kind}"))
+        # default value
+        if source is not SettingValueStore.DEFAULT:
+            default_value = setting_def.value
+            if default_value is not None:
+                details.add_row(Text("Default"), Text(f"{default_value!r}"))
+        # env vars
+        env_keys = [
+            var.definition for var in self.settings.setting_env_vars(setting_def)
+        ]
+        details.add_row(Text("Env(s)"), Text(f"{', '.join(env_keys)}"))
+        # setting description (markdown)
+        post = []
+        if setting_def.description:
+            post.append(
+                Group(
+                    Text(" Description:"),
+                    Panel(Markdown(setting_def.description, justify="left")),
+                )
+            )
+        # docs url
+        docs_url = self.settings.docs_url
+        if docs_url:
+            post.append(
+                Text.from_markup(
+                    f" To learn more about {self.settings.label} and its settings, visit [link={docs_url}]{docs_url}[/link]"
+                )
+            )
+        self.console.print(Panel(Group(*pre, details, *post)))
+
+    def _value_prompt(self, config_metadata):
+        if config_metadata["setting"].kind != SettingKind.OPTIONS:
+            return (
+                click.prompt(
+                    "New value:",
+                    default="",
+                    show_default=False,
+                    hide_input=True,
+                    confirmation_prompt=True,
+                )
+                if config_metadata["setting"].is_redacted
+                else click.prompt("New value", default="", show_default=False)
+            )
+
+        options_index = {
+            str(index + 1): value
+            for index, value in enumerate(
+                (chs["label"], chs["value"])
+                for chs in config_metadata["setting"].options
+            )
+        }
+
+        for index, value in options_index.items():
+            click.echo(f"{index}. {value[0]}")
+        click.echo()
+        chosen_index = click.prompt(
+            "Select value:",
+            type=click.Choice(list(options_index.keys())),
+            show_default=False,
+        )
+        return options_index[chosen_index][1]
+
+    def configure(
+        self, name, index=None, last_index=None, show_set_prompt=True, is_secret=False
```

Comment:
```suggestion
        self, name, index=None, last_index=None, show_set_prompt=True
```

Think `is_secret` wasn't actually used.